apps/tailwind-components/components/input/Select.vue

```diff
@@ -1,11 +1,13 @@
 <script setup lang="ts">
-import type { N } from "vitest/dist/reporters-MmQN-57K.js";
+interface ISelectOptions {
+  value: string | number;
+  label?: string;
+}
```

Comment:
As I currently am building a screen where I want to use this component, I would like to insert objects with label as one of its fields. Would you be against adding this?